models/return_data_object.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+"""
+module to handle all incoming requests for server side information
+"""
+import json
+
+
+class ReturnDataTransferObject:
+    """
+    contains information and serialisation for object.
+    """
+
+    def __init__(self):
+        """how to initialize returnDTO"""
+        self.__objectInformationDictionary = {}
```

Comment:
Any pros about using the instance member here instead of a getter / setter like the others ?

Reply:
only because I'm not setting it here, just instansiating it. The actual 'setting' of the object is done in the background of other methods (not clear I know but was just the simple way to do it now). When another object is updated, this is added to this information Dictionary.
There is a 'getter' method for it however.